### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by juajum**
[DEMO LINK](https://juajum.github.io/react_movies-list-add-form/)

### CODE REVIEW

**Review by viktorlyt — APPROVED** [review]
good job!
but it would be better to add placeholder to every input

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,16 +1,157 @@
...
-type State = {};
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
 
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
+export const NewMovie: React.FC<Props> = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
```
Comment (Olexandr6) [inline src/components/NewMovie/NewMovie.tsx:10]: use:  `is, check, should,` etc. for boolean variables
```suggestion
  const [isTitleError, setTitleError] = useState(false);
```

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,16 +1,157 @@
...
-        Put the form here
-      </form>
-    );
+  function submit(event: React.FormEvent) {
+    event.preventDefault();
+    setTitleError(!title);
+    setDescriptionError(!description);
+    setImgUrlError(!imgUrl);
+    setImdbUrlError(!imdbUrl);
+    setImdbIdError(!imdbId);
```
Comment (Olexandr6) [inline src/components/NewMovie/NewMovie.tsx:26]: I can't understand why do you do this here?

**Review by Olexandr6 — APPROVED** [review]
Pay attention to the comments

**Pull request closed without merging**